runtime/doc/index.txt

```diff
@@ -1362,7 +1362,6 @@ tag	      command	      action ~
 |:nunmap|	:nun[map]	like ":unmap" but for Normal mode
 |:nunmenu|	:nunme[nu]	remove menu for Normal mode
 |:oldfiles|	:ol[dfiles]	list files that have marks in the viminfo file
-|:open|		:o[pen]		start open mode (not implemented)
 |:omap|		:om[ap]		like ":map" but for Operator-pending mode
```

Comment:
One these gets the `:o` as its short form since `:open` no longer has it. (I'm not sure which)